tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,19 @@
+def countDigits(n):
```

Comment:
it looks like JS, but python has another code style. You can read from (here)[https://www.python.org/dev/peps/pep-0008/]

For example: `count_digits(n)`, but we need use: `count_tens`